package.json

```diff
@@ -20,8 +20,6 @@
   },
   "homepage": "https://github.com/kjmatt91/tdd-strict-equal#readme",
   "devDependencies": {
-    "chai": "^4.3.4",
-    "eslint": "^7.29.0",
```

Comment:
you can keep these in the package.json. but just not use them in the test files it self